### PR TITLE
[Doc Improvement] add note to validDomains manifest reference

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Add a note to "validDomains" reference about wildcards not honored in Office Add-ins.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/199a3132018ade20087b820502050c134c1bc897/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 91 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/811572e8-959b-46ba-9537-7daa42991209) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 1]
```diff
@@ -566,6 +566,9 @@ Specifies the native features on a user's device that your app may request acces
 
 A list of valid domains from which the app expects to load any content. Domain listings can include wildcards, for example `*.example.com`. The valid domain matches exactly one segment of the domain; if you need to match `a.b.example.com` then use `*.*.example.com`. If your tab configuration or content UI needs to go to any other domain besides the one use for tab configuration, that domain must be specified here.
 
+> [!NOTE]
+> Domains that contain a wildcard are ignored by Office Add-ins (configured with an `extensions` property in the manifest). If your app includes and Office Add-in, you must spell out the complete domain name for domains that the add-in is going to access.
```
Comment (surbhigupta12) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:570]:
```suggestion
> Office Add-ins, configured with an `extensions` property in the manifest, disregard domains containing a wildcard. If your app incorporates an Office Add-in, ensure to specify the full domain name for the domains that the add-in will access.
```

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]

**Comment by acrolinx-at-msft2** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e422347a656d6af0c3be1d92810bd65ddfe2a030/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 91 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d9ffc6e-248e-495b-9766-4efcf54116d2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 2]
```diff
@@ -566,6 +566,9 @@ Specifies the native features on a user's device that your app may request acces
 
 A list of valid domains from which the app expects to load any content. Domain listings can include wildcards, for example `*.example.com`. The valid domain matches exactly one segment of the domain; if you need to match `a.b.example.com` then use `*.*.example.com`. If your tab configuration or content UI needs to go to any other domain besides the one use for tab configuration, that domain must be specified here.
 
+> [!NOTE]
+> Office Add-ins, configured with an `extensions` property in the manifest, disregard domains containing a wildcard. If your app incorporates an Office Add-in, ensure to specify the full domain name for the domains that the add-in will access.
```
Comment (Rick-Kirkham) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:570]:
```suggestion
> Office Add-ins, configured with an `extensions` property in the manifest, disregard domains containing a wildcard. If your app incorporates an Office Add-in, specify the full domain name for the domains that the add-in will access.
```

**Comment by acrolinx-at-msft2** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eddc106baf30e611cb7138f40251df35d3742589/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 91 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d01fd8bc-249b-4d59-9fa0-1baeb3d1de5e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fa6f8fccc894806cded17160839e822d6f411ede/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 91 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8e07b87b-cbbd-4d2f-a930-1b8ba7df826b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**